src/main/java/com/epam/dao/UserDao.java

```diff
@@ -0,0 +1,121 @@
+package com.epam.dao;
+
+import com.epam.entities.User;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVWriter;
+import com.opencsv.bean.CsvToBean;
+import com.opencsv.bean.HeaderColumnNameTranslateMappingStrategy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by Almas_Doskozhin
+ * on 4/2/2016.
+ */
+public class UserDao {
```

Comment:
Looks like your Persistence layer has too many responsibilities.
Keep in mind, that it must be responsible only for saving data in database